app/Http/Controllers/MonitorController.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+require_once 'include/common.php';
```

Comment:
This require isn't necessary since `AbstractController` includes it for us already here:
https://github.com/Kitware/CDash/blob/5088cddc8e654f2760288b17662d4852e3016526/app/Http/Controllers/AbstractController.php#L12